jcl/src/jdk.management/share/classes/com/ibm/lang/management/RuntimeMXBean.java

```diff
@@ -83,7 +83,15 @@ public String idleStateName() {
 	 * 
 	 * @return A long representing the process ID (pid) on the underlying 
 	 * operating system.
+	/*[IF Java10]
+	 * 
+	 * @deprecated(forRemoval = true, since = "10") Use 
+	 * {@link java.lang.management.RuntimeMXBean#getPid() getPid()} instead.
+	 */
+	@Deprecated
+	/*[ELSE]
 	 */
+	/*[ENDIF]*/
 	public long getProcessID();
```

Comment:
This interface method is not necessary for `Java 10` after `java/lang/management/RuntimeMXBean` adding it which is inherited by this interface class.